src/v2i-hub/TelematicBridgePlugin/manifest.json

```diff
@@ -11,6 +11,31 @@
             "key": "LogLevel",
             "default": "INFO",
             "description": "The log level for this plugin"
+        },
+        {
+            "key": "NATSUrl",
+            "default": "nats://127.0.0.1:4222",
+            "description": "The log level for this plugin"            
+        },
+        {
+            "key": "UnitId",
+            "default": "v2xhub_id",
```

Comment:
We are currently, in CDASim pulling and infrastructure ID from an environment variable. You may want to do the same here. See CDASimAdapter

Reply:
This is not the same concept as the infrastructure id. It is a unique identifier we give to a telematic unit to distinguish it from other units like streets and vehicle. Is there any rule when we use the UI configuration parameter and when to use the environment variables? All these parameters make sense when they are together.

Reply:
Hmm, so part of the UC for this plugin is to disable health of different RSUs, and therefore infrastructure instances. Coming up with and ID that is completely different from the J2735 intersection ID and our infrastructure ID I think is bad practice. Then we will have 3 different IDs for the same instance, all which are completely indepedent.

Reply:
I think this unit id value can be anything that makes sense to end user during their testing with v2xhub as long as it is unique in the telematic tool.

Reply:
So lets tell the user to use intersection name from MAP message if available.

Reply:
Something like <rsu_21312>. We have not standardized this yet but I think if we say it has to be rsu_<map-intersection-id> then maybe we can minimize different ID

Reply:
The unit id is under the telematic tool context. They may share the same value. For debugging and ease of usability purpose, it might be more obvious if we keep both unit name and unit id in the same place like here on the V2xHub UI.